source/elements/oneMKL/source/domains/spblas/operations/spmv.rst

```diff
@@ -0,0 +1,378 @@
+           oneapi::mkl::sparse::dense_vector_handle_t x_handle,
+           void*                                      beta,
+           oneapi::mkl::sparse::dense_vector_handle_t y_handle,
+           oneapi::mkl::sparse::spmv_alg              alg,
+           oneapi::mkl::sparse::spmv_descr_t          spmv_descr,
+           const std::vector<sycl::event>             &dependencies = {},
+       );
+
+   }
+
+.. container:: section
+
+   .. rubric:: Notes
+
+   - ``spmv_buffer_size`` and ``spmv_optimize`` must be called at least once before ``spmv``
+     with the same arguments. ``spmv`` can then be called multiple times.
+   - The data of the handles can be reset-ed before each call to ``spmv``.
+   - ``spmv_optimize`` and ``spmv`` are asynchronous.
+   - The algorithm defaults to ``spmv_alg::default_alg`` if a backend does not
+     support the provided algorithm.
+
+   .. rubric:: Input Parameters
+
+   queue
+      The SYCL command queue which will be used for SYCL kernels execution.
+
+   opA
+      Specifies operation ``op()`` on the input matrix. The possible options are
+      described in :ref:`onemkl_enum_transpose` enum class.
+
+   alpha
+      Host or USM pointer representing :math:`\alpha`. The USM allocation can be
+      on the host or device. Must be of the same type than the handles' data
+      type.
+
+   A_view
+      Specifies which part of the handle should be read as described by
+      :ref:`onemkl_sparse_matrix_view`.
+
+   A_handle
+      Sparse matrix handle object representing :math:`A`.
+
+   x_handle
+      Dense vector handle object representing :math:`x`.
+
+   beta
+      Host or USM pointer representing :math:`\beta`. The USM allocation can be
+      on the host or device. Must be of the same type than the handles' data
+      type.
+
+   y_handle
+      Dense vector handle object representing :math:`y`.
+
+   alg
+      Specifies the :ref:`spmv algorithm<onemkl_sparse_spmv_alg>` to use.
+
+   spmv_descr
+      Initialized :ref:`spmv descriptor<onemkl_sparse_spmv_descr>`.
+
+   temp_buffer_size
+      Output buffer size in bytes.
+
+   workspace
+      | Workspace buffer or USM pointer, must be at least of size
+        ``temp_buffer_size`` bytes and the address aligned on the size of the
+        handles' data type.
+      | If it is a buffer, its lifetime is extended until the :ref:`spmv
+        descriptor<onemkl_sparse_spmv_descr>` is released. The workspace cannot
+        be a sub-buffer.
+      | If it is a USM pointer, it must not be free'd until ``spmv`` has
+        completed. The data must be accessible on the device.
+
+   dependencies
+      List of events to depend on before starting asynchronous tasks that access
+      data on the device. Defaults to no dependencies.
+
+.. container:: section
+
+   .. rubric:: Output Parameters
+
+   temp_buffer_size
+      Output buffer size in bytes. A temporary workspace of this size must be
+      allocated to perform the specified spmv.
```

Comment:
```suggestion
      Output buffer size in bytes. A temporary workspace of at least this size
      must be allocated to perform the specified spmv.
```

Reply:
Done in https://github.com/oneapi-src/oneAPI-spec/pull/522/commits/1cbd268c6c39aa2f1464ab5df2cd6e24f6554c1e